### PR TITLE
Add activities when space is achieved or unachieved

### PR DESCRIPTION
**Pull request by haykhakobyan94**

### CODE REVIEW

**Comment by luke-** [comment]
@danielkesselberg Thanks, what is the best way to check this locally?

File: protected/humhub/modules/space/activities/views/spaceUnArchieved.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+use yii\helpers\Html;
+use humhub\libs\Helpers;
+
+echo Yii::t('ActivityModule.views_activities_ActivitySpaceMemberAdded', '%spaceName% has been unarchived', [
+    '%spaceName%' => '<strong>' . Html::encode(Helpers::truncateText($source->name, 40)) . '</strong>'
+]);
+
+?>
+<br/>
+
```
Comment (danielkesselberg) [inline protected/humhub/modules/space/activities/views/spaceUnArchieved.php]: Do you need this blank lines?

File: protected/humhub/modules/space/activities/views/spaceArchieved.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+use humhub\libs\Helpers;
+
+echo Yii::t('ActivityModule.views_activities_ActivitySpaceMemberAdded', '%spaceName% has been archived', [
+    '%spaceName%' => '<strong>' . Html::encode(Helpers::truncateText($source->name, 40)) . '</strong>'
+]);
+
+?>
+<br/>
+
+
```
Comment (danielkesselberg) [inline protected/humhub/modules/space/activities/views/spaceArchieved.php]: Do you need this blank lines?

**Comment by danielkesselberg** [comment]
@luke- I installed php code sniffer with composer like described here: https://github.com/squizlabs/PHP_CodeSniffer

**Pull request merged**